eng/WpfArcadeSdk/tools/CreateTestPayload.targets

```diff
@@ -0,0 +1,49 @@
+<Project>
+    <!-- We use a local RID here since we don't want to change the build layout by setting the actual RID. -->
+  <PropertyGroup>
+    <LocalRuntimeIdentifier Condition="'$(Platform)'=='x64' or '$(Platform)'=='x86' or '$(Platform)'=='arm' or '$(Platform)'=='arm64'">win-$(Platform)</LocalRuntimeIdentifier>
+    <LocalRuntimeIdentifier Condition="'$(Platform)'=='Win32' or '$(Platform)'=='AnyCPU'">win-x86</LocalRuntimeIdentifier>
+    <NoWarn>$(NoWarn);NU3027;NU3018</NoWarn>
+
+     <!-- Arcade SDK defaults to AnyCpu ~= x64. In WPF, we will map AnyCpu => x86 instead -->
+    <PlatformFolder>x86</PlatformFolder>
+    <PlatformFolder Condition="'$(Platform)' == 'x64'">x64</PlatformFolder>
+    <PublishRoot>$(RepoRoot)artifacts\test\$(Configuration)\$(PlatformFolder)</PublishRoot>
+
+    <WpfTestBasePublishPath>$(PublishRoot)\Test</WpfTestBasePublishPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="runtime.$(LocalRuntimeIdentifier).Microsoft.DotNet.Wpf.Test" Version="$(MicrosoftDotNetWpfTestPackageVersion)" GeneratePathProperty="true" />
+  </ItemGroup>
+
+  <PropertyGroup Condition="'$(LocalRuntimeIdentifier)'=='win-x86'">
+    <MicrosoftDotNetWpfTestPath>$(Pkgruntime_win-x86_Microsoft_DotNet_Wpf_Test)\tools\$(LocalRuntimeIdentifier)\Test\</MicrosoftDotNetWpfTestPath>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(LocalRuntimeIdentifier)'=='win-x64'">
+    <MicrosoftDotNetWpfTestPath>$(Pkgruntime_win-x64_Microsoft_DotNet_Wpf_Test)\tools\$(LocalRuntimeIdentifier)\Test\</MicrosoftDotNetWpfTestPath>
+  </PropertyGroup>
+
+  <Target Name="CreateTestPayload" BeforeTargets="Test" AfterTargets="Build">
+    <ItemGroup>
+      <MicrosoftDotNetWpfTestContents Include="$(MicrosoftDotNetWpfTestPath)\**\*.*" />
```

Comment:
Extra backslash here after the path var.  Generally, path vars will always include a trailing backslash so you shouldn't add one during use.

Reply:
I'd scrub this file for cases where this occurs

Reply:
Just spot checked some other targets and we generally need a scrub of that, oops.